Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/ModelTime.mo

```diff
@@ -0,0 +1,51 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.SetPoints;
+block ModelTime "Model time"
```

Comment:
This should rather be called 'StandardTime' as I don't think it is meaningful to talk about 'ModelTime' in a building automation system.